users/views.py

```diff
@@ -1,3 +1,72 @@
-from django.shortcuts import render
+import json
+import re
+import bcrypt
+import jwt
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from my_settings import SECRET_KEY
```

Comment:
`my_settings.py`가 아니라 django settings.py에서 불러와서 사용하도록 수정해주세요~

```suggestion
from my_settings import SECRET_KEY
```

Reply:
from django.conf import settings 로 바꾸고 
74번째줄 토큰 생성시 시크릿키 settings에서 불러와서 생성할 수 있도록 수정했습니다.